src/dotnet-interactive/Program.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.DotNet.Interactive.App
 {
     public class Program
     {
+        private static Random R = new Random();
```

Comment:
This can be moved inside `GetFreePort` so it's only instantiated if needed.

Reply:
Yup it can :)